### ISSUE TITLE
View one stashtab

### ISSUE THREAD

**Issue by Davst** [viktorgullmark/exilence-next#662]
**Is your feature request related to a problem? Please describe.**
Usually I have my items in a set of stashtabs where I shift prices lower and then dump whatever is left over in the lowest priced one. I would like to use Exilence when I do dump these items, I'd like to pull up that individual stash tab and be able to see what exilence perceives to be the highest value items to see if there might be a few items I'd want to hold on to anyway.

**Describe the solution you'd like**
The possibility to quick select one tab temporarially, without having to do the whole add/remove all the tabs in the filter. 

**Describe alternatives you've considered**
As an alternative a dropdown that had something like display: All tabs or just selecting one would work

**Additional context**

**Comment by viktorgullmark** [comment]
Is this what youre looking for? We have a filter option to the right of the table, not really obvious though :D

![image](https://user-images.githubusercontent.com/11405940/116300938-c6b01800-a79f-11eb-9da0-bfa73d892006.png)

**Comment by Davst** [comment]
Almost. I have like 9 tabs followed. Which means I’d have to remove 8 then re add them. As I suggested. It’d be nice to be able to click one of those in your screenshot to temp show just the content of that tab without changing the filter. 

Looking for something that doesn’t require 10+ clicks

**Comment by viktorgullmark** [comment]
Gotcha, thanks for the explanation and sorry I was abit quick on the ball :) Will see what we can do about this, although low on the prio since we have a bunch of other more critical issues to tend to sadly..

**Comment by Davst** [comment]
Absolutely, just pointing out a possible improvement. I can understand that this might be down the line =)

**Comment by sbsrnt** [comment]
@Davst possible way to get that rolling now is to set couple profiles with one tab that you want to follow. Initial setup may be daunting though.
